pod-operation/src/components/lim_temperature.rs

```diff
@@ -6,6 +6,19 @@ use ads1x1x::{Ads1x1x, DynamicOneShot, SlaveAddr};
 use nb::block;
 use rppal::i2c::I2c;
 
+const DIVIDER_RESISTANCE: f32 = 22000.0;
+const VCC: f32 = 5.0;
+const BETA: f32 = 3950.0;
+const R_0: f32 = 10000.0;
+const ROOM_TEMP: f32 = 298.15;
```

Comment:
Suggestion: although most of these values are self explanatory, for improving understanding, could we label these with the corresponding units as inline comments?

Reply:
I meant particularly the units of the values (ohms, volts)

Reply:
Should I provide the units for `BETA` as well or leave it as is? Seems like the units would be complex for that one.

Reply:
Oh wait, seems like `BETA` is just in Kelvins, if my dimensional analysis is correct.